Revit_Core_Engine/Query/LookupParameter.cs

```diff
@@ -62,6 +62,29 @@ public static Parameter LookupParameter(this Element element, IEnumerable<string
             return null;
         }
 
+        [Description("Searches a Revit Family Instance for the first existing parameter of a given name, by first looking for Instance parameters and then, optionally, for Type parameters.")]
+        [Input("element", "Revit element to be queried.")]
+        [Input("parameterName", "Names of the parameter to be iterated over in search for the parameter.")]
+        [Input("allowTypeParameters", "Optional, whether or not to also look for Type parameters if no Instance parameters were found.")]
+        [Output("parameter", "Parameter extracted from the input Revit Family Instance.")]
+        public static Parameter LookupParameter(this Element element, string parameterName, bool allowTypeParameters = true)
```

Comment:
Apologies for another change request, but could we please not set the default to `allowTypeParameters`? It is pretty crucial to the output while it can easily be confused with the native Revit API `Element.LookupParameter`...
```suggestion
        public static Parameter LookupParameter(this Element element, string parameterName, bool allowTypeParameters)
```